<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="leftFrameTool">
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionbc"/>
   <addaction name="actionsx"/>
   <addaction name="actionzz"/>
   <addaction name="actionReset"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionRatation">
   <property name="text">
    <string>Ratation</string>
   </property>
  </action>
  <action name="actionCur">
   <property name="text">
    <string>Cur</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/icon/zoomIn.png</normaloff>:/image/icon/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/icon/zoomOut.png</normaloff>:/image/icon/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionzz">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/icon/drag.png</normaloff>:/image/icon/drag.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionsx">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/icon/drop.png</normaloff>:/image/icon/drop.png</iconset>
   </property>
   <property name="text">
    <string>sx</string>
   </property>
  </action>
  <action name="actionbc">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/icon/saveproject.png</normaloff>:/image/icon/saveproject.png</iconset>
   </property>
   <property name="text">
    <string>bc</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/image/icon/openproject.png</normaloff>:/image/icon/openproject.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
